hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -1453,6 +1456,22 @@ message PartInfo {
     required uint64 size = 4;
 }
 
+/**
+ Stores the raw payload of rpc request, used in OmRPCLoadGenerator
```

Comment:
```suggestion
 Rpc request with optional payload, used in OmRPCLoadGenerator
```

Reply:
make sense, will change the wording in other place like this.